test/any/parsing/GeckoParser-test.js

```diff
@@ -40,4 +40,14 @@ describe('GeckoParser', function() {
     `);
     expect(p.value).toBeDefined();
   });
+  it('should report parse complete status to console', function() {
```

Comment:
These should be moved to a new `BaseParser-test.js`. The tests can `beforeEach()` create a concrete parser class with a simple parser in it for the purpose of testing `logParse()`.